consensus/src/network_tests.rs

```diff
@@ -298,18 +348,36 @@ impl NetworkPlayground {
         matches!(&msg.1, ConsensusMsg::EpochChangeProof(_))
     }
 
-    fn is_message_dropped(&self, src: &Author, net_req: &PeerManagerRequest) -> bool {
+    pub fn extend_author_to_twin_ids(&mut self, author: Author, twin_id: TwinId) {
```

Comment:
I couldn't find a way to do this in 'add_node' because it calls the async function 'start_node_outbound_handler' to which we provide a fully populated copy of 'author_to_twin_ids'.

Reply:
just pass the Arc<Mutex<HashMap<_>>> to `start_node_outbound_handler` and we can do this inside `add_node`.

Reply:
Sounds good, thanks. Done.